src/test/java/wooteco/subway/admin/acceptance/LineStationAcceptanceTest.java

```diff
@@ -23,28 +40,111 @@ void setUp() {
     public static RequestSpecification given() {
         return RestAssured.given().log().all();
     }
-
-    /**
-     *     Given 지하철역이 여러 개 추가되어있다.
-     *     And 지하철 노선이 추가되어있다.
-     *
-     *     When 지하철 노선에 지하철역을 등록하는 요청을 한다.
-     *     Then 지하철역이 노선에 추가 되었다.
-     *
-     *     When 지하철 노선의 지하철역 목록 조회 요청을 한다.
-     *     Then 지하철역 목록을 응답 받는다.
-     *     And 새로 추가한 지하철역을 목록에서 찾는다.
-     *
-     *     When 지하철 노선에 포함된 특정 지하철역을 제외하는 요청을 한다.
-     *     Then 지하철역이 노선에서 제거 되었다.
-     *
-     *     When 지하철 노선의 지하철역 목록 조회 요청을 한다.
-     *     Then 지하철역 목록을 응답 받는다.
-     *     And 제외한 지하철역이 목록에 존재하지 않는다.
-     */
     @DisplayName("지하철 노선에서 지하철역 추가 / 제외")
     @Test
     void manageLineStation() {
```

Comment:
이 피드백 반영 안하신 것 같아요 😓

Reply:
```suggestion
    void manageLineStation() {
        //Given
        createStation("잠실");
        createStation("천호");
        createStation("몽촌토성");
        //And
        createLine("8호선");

        //When
        createLineStation(1L, null, 1L);
        createLineStation(1L, 1L, 2L);
        createLineStation(1L, 2L, 3L);
```
이런 식으로 주석을 달아 줬는데, 이것 말고 `given().` 등을 사용해야 하나요?